src/test/shared/telemetry/telemetryEvent.test.ts

```diff
@@ -59,6 +59,37 @@ describe('TelemetryEventArray', () => {
             assert.deepStrictEqual(data[0].Metadata, undefined)
         })
 
+        it('Rejects entries that have null value or MetricName', () => {
```

Comment:
It would be ideal to test these inputs independently. We've been burned on similar tests before, where the test passes with a false positive.

Reply:
Split them